frontend/src/components/Icons.module.css

```diff
@@ -0,0 +1,45 @@
+.button_primary {
+background-color: ion-color-primary;
```

Comment:
wrong indentation

Reply:
and why is button-styling for non-icon buttons within the Icons.module.css?

Reply:
You're right, now that I think about it. Renamed to Buttons.module.css

Reply:
Moved all CSS into RoutePlanner.module.css and completely removed the Icons/Buttons.module.css